bandit/core/manager.py

```diff
@@ -23,6 +24,8 @@
 
 
 LOG = logging.getLogger(__name__)
+NOSEC_SPECIFIC_IDS_IGNORE = re.compile(r"([bB][\d]+),?[ ]?")
+NOSEC_SPECIFIC_NAMES_IGNORE = re.compile(r"([a-z_]+),?[ ]?")
```

Comment:
Do we need to have two distinct regular expressions here? In fact, do we need to have them be case sensitive like this if we even want them like this?

I'm thinking we could either do:

```py
r"(?:(B\d+|[a-z_]+),?)+"
```

If we're concerned about case-sensitivity (in which case we should only accept `B123` and `check_name`, or we compile it with `re.IGNORE`. This has the downside of letting folks mix and match short code and check name, but I'm also not super fond of long check names in the ignore statement. Either way, allowing only one or another on different lines will definitely lead to frustration by folks